### ISSUE TITLE
uTox auto closes when I try create a group chat

### ISSUE THREAD

**Issue by Z33DD** [uTox/uTox#1411]
### Overview

1. uTox version: 0.17.0
2. Operating system: Pop!_OS 19.10 x86_64 (based on Ubuntu 19.10)
3. Desktop Environment/Window Manager (Linux/BSD only): GNOME (Pop!_OS flavor)

### Issue

uTox auto closes when I try create a group chat.
In terminal, I get this:
```
❯ utox
Gtk-Message: 20:45:26.579: Failed to load module "appmenu-gtk-module"
XLib Tray:    Incoming tray window event (28)
XLib tray:    Reached end of function, this is bad juju!
XLib Event:   Type UTF8_STRING || Prop TARGETS 
Chatlog:      Can't malloc that much, you'll probably have to move or delete your history for this peer.
                Friend number 53176AF8A86D250A63F25FE5EB1E4F76C67401385699CC08375AF31101E1B608, count 5, actual_count 1, start at 0, error size 7382074271468577824.

FileTransfer: Can't decon a FT that doesn't exist!
get_group:     index: 1 is out of bounds.
get_group:     index: 1 is out of bounds.


F-List:       :Selected group no longer exists. Group number: 1
```

**Comment by avoidr** [comment]
Probably the same thing as #1408.

**Comment by avoidr** [comment]
Are you able to test uTox v0.17.2? I believe I've fixed it in #1439.

**Comment by avoidr** [comment]
Seems dupe of #1437.
